src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const list = document.getElementsByTagName('li');
+
+function getEmployees(listProperty) {
```

Comment:
you take `listProperty` in parameters, but you don't use it?

Reply:
Fixed. Add parametr